x-pack/plugin/runtime-fields/src/test/java/org/elasticsearch/xpack/runtimefields/mapper/RuntimeFieldMapperTests.java

```diff
@@ -376,14 +376,7 @@ protected Object buildScriptFactory(ScriptContext<?> context) {
                 };
 
                 public Set<ScriptContext<?>> getSupportedContexts() {
-                    return Set.of(
-                        BooleanFieldScript.CONTEXT,
-                        DateFieldScript.CONTEXT,
-                        DoubleFieldScript.CONTEXT,
-                        IpFieldScript.CONTEXT,
-                        StringFieldScript.CONTEXT,
-                        LongFieldScript.CONTEXT
-                    );
```

Comment:
👍

Reply:
No they can't because the plugin class is in a different package now :)